packages/story-editor/src/app/story/useStoryReducer/actions.js

```diff
@@ -27,8 +27,8 @@ const addPage =
 
 const addPageAt =
   (dispatch) =>
-  ({ page, position }) =>
-    dispatch({ type: types.ADD_PAGE, payload: { page, position } });
+  ({ page, position, select }) =>
+    dispatch({ type: types.ADD_PAGE, payload: { page, position, select } });
```

Comment:
Why is this needed?

Reply:
I should rename that ...

This is whether we want to select the page i.e. have focus move to it or not --- but default it will select / focus 

We don't want that in this case --- we want to create a page and not move to it.